core-web/libs/sdk/client/src/lib/client/content/builders/collection/collection.ts

```diff
@@ -237,12 +272,19 @@ export class CollectionBuilder<T = unknown> {
     }
 
     /**
-     * Takes a number that represents the depth of the relationships of a content
-     *
-     * The `depth` is set to 0 by default and the max supported value is 3.
+     * Sets the depth of the relationships of the content.
```

Comment:
Add a link to the doc where it explains depth.

Reply:
Done!